operator/redisfailover/service/generator.go

```diff
@@ -281,6 +281,196 @@ func generateHAProxyService(rf *redisfailoverv1.RedisFailover, labels map[string
 	}
 }
 
+func generateHAProxyRedisSlaveDeployment(rf *redisfailoverv1.RedisFailover, labels map[string]string, ownerRefs []metav1.OwnerReference) *appsv1.Deployment {
+
+	name := rf.GenerateName(redisHAProxySlaveRedisName)
```

Comment:
Can we add name generators for HAProxy resources in `names.go` and follow the name generation conventions therein?

Reply:
moved to names generator